staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/types.go

```diff
@@ -68,20 +70,23 @@ type CustomResourceDefinitionVersion struct {
 
 // CustomResourceColumnDefinition specifies a column for server side printing.
 type CustomResourceColumnDefinition struct {
-	// name is a human readable name for the column.
+	// Name is a human readable name for the column.
```

Comment:
revert this... comments should follow the serialized field names

Reply:
> revert this... comments should follow the serialized field names (applies to all)

will do, though I always thought exported field names need to have comments starting with an uppercase letter to be present in the godoc...is not right? :thinking:

Reply:
> (applies to all)

did you mean for all existing comments as well?